schema.sql

```diff
@@ -91,4 +97,51 @@ CREATE TABLE visits (
     animal_id INT NULL REFERENCES animals(id) ON DELETE CASCADE,
     vet_id INT NULL REFERENCES vets(id) ON DELETE CASCADE,
     date_of_visit DATE NOT NULL
-);
\ No newline at end of file
+);
+*/
+
+-- Week 2
+--Monday
+
+CREATE TABLE owners (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    full_name varchar(100) NOT NULL,
+    age INT NULL
+);
+
+CREATE TABLE species (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name varchar(100) NOT NULL
+);
+
+CREATE TABLE animals (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name varchar(100) NOT NULL,
+    species_id INT NULL REFERENCES species(id) ON DELETE CASCADE,
+    owner_id INT NULL REFERENCES owners(id) ON DELETE CASCADE,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT NOT NULL,
+    neutered BOOLEAN NOT NULL,
+    weight_kg DECIMAL NOT NULL
+);
+
+CREATE TABLE vets (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name varchar(100) NOT NULL,
+    age INT NOT NULL,
+    date_of_graduation DATE NOT NULL
+);
+
+CREATE TABLE visits (
+    animal_id INT NULL REFERENCES animals(id) ON DELETE CASCADE,
+    vet_id INT NULL REFERENCES vets(id) ON DELETE CASCADE,
+    date_of_visit DATE NOT NULL
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Create index to enhance performance
+CREATE INDEX visits_animal_id ON visits(animal_id);
+CREATE INDEX visits_vet_id ON visits(vet_id);
+CREATE INDEX owner_email ON owners(email);
```

Comment:
- [ ] Kindly make sure that the execution time of each query before creating the indexes is at least 1000ms, as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=If%20you%20get%20Execution%20time%3A%20X%20ms%20and%20X%20%3C%201000%3A%20please%20go%20back%20to%20point%203.%20and%20repeat%20until%20you%20get%20a%20value%20bigger%20than%201000ms) requirement.

Reply:
@ShadyShawkat we have attached the screenshots for the 'BEFORE' creating the index Are you referring to the 2 pictures we have attached

Reply:
@michael-duke, yes the 'BEFORE` images should have execution times for each query that surpasses 1000ms.

Reply:
Thanks @ShadyShawkat  we will adjust and make a request, happy coding 😄 🚀